pkg/network/config/config_linux_test.go

```diff
@@ -33,3 +38,42 @@ func TestDisableRootNetNamespace(t *testing.T) {
 	defer ns.Close()
 	require.True(t, ns.Equal(rootNs))
 }
+
+func TestEventStreamEnabledForSupportedKernelsLinux(t *testing.T) {
+	t.Run("for kernels <4.15.0", func(t *testing.T) {
+		if runtime.GOOS != "linux" {
+			t.Skip("This is only for linux")
+		}
+		kv, err := ebpfkernel.NewKernelVersion()
+		kv4150 := kernel.VersionCode(4, 15, 0)
+		require.NoError(t, err)
+		if kv.Code >= kv4150 || kv.IsRH8Kernel() {
+			t.Skip("This test should only be run on kernels < 4.15.0 and RH7")
+		}
+		aconfig.ResetSystemProbeConfig(t)
+		t.Setenv("DD_SYSTEM_PROBE_EVENT_MONITORING_NETWORK_PROCESS_ENABLED", strconv.FormatBool(true))
+
+		cfg := aconfig.SystemProbe
+		sysconfig.Adjust(cfg)
+
+		require.False(t, cfg.GetBool("event_monitoring_config.network_process.enabled"))
+	})
+	t.Run("for kernels >=4.15.0 with default value", func(t *testing.T) {
+		if runtime.GOOS != "linux" {
+			t.Skip("This is only for linux")
+		}
+		kv, err := ebpfkernel.NewKernelVersion()
+		kv4150 := kernel.VersionCode(4, 15, 0)
+		require.NoError(t, err)
```

Comment:
nit (as always with nits, these don't need to be addressed if you disagree or need to merge) this test is re-implementing the logic in the function which it's testing.

I think a nicer way to test this change would be to run both:

* the constructor for new process event data stream
* DisablePESConfig(true)

If the constructor passes, but DisablePESConfig is true, the test fails.

If the constructor fails, the DisablePESConfig is false, the test fails.

Otherwise the test passes.